services/web-server/README.md

```diff
@@ -271,3 +271,16 @@ Variables:
 ## Data Flow Diagram
 
 ![data flow](https://cldup.com/e3lrkf28ab.png)
+
+## Style Guide
+
+### Contents
+
+* [Schema Field Definitions](#schema-field-definitions)
+ 
+### Schema Field Definitions
+
+* For fields that may trigger an additional request for a client,
+add a comment above the field definition of the form "Note: This field will trigger an additional request."
+This will make sure developers are aware of possible performance penalties.
+
```

Comment:
It would probably make sense to put this information with the other best-practices in `dev-docs` in the root.